crates/sui-json/src/lib.rs

```diff
@@ -391,6 +391,9 @@ fn is_primitive_type_tag(t: &TypeTag) -> bool {
             if resolved_struct == RESOLVED_SUI_ID {
                 return true;
             }
+            else if resolved_struct == RESOLVED_SUI_UID {
```

Comment:
nit: can you combine it with line 391 to become `if resolved_struct == RESOLVED_SUI_ID || resolved_struct == RESOLVED_SUI_UID`?